db/migrate/20231103165752_add_hmis_activity_log.rb

```diff
@@ -0,0 +1,19 @@
+class AddHmisActivityLog < ActiveRecord::Migration[6.1]
+  def change
+    create_table :hmis_activity_logs do |t|
+      t.references :user, null: false, index: true
+      t.references :data_source, null: false, index: false
+      t.jsonb :resolved_fields
+      t.string :ip_address, null: false
+      t.string :session_hash
+      t.jsonb :variables, comment: 'GraphQL variables'
+      t.string :referer, comment: 'user-provided'
+      t.string :operation_name, comment: 'user-provided GraphQL operation name'
```

Comment:
This is user-provided, but it can be trusted, right? It's definitely the operation that was performed, and if it's not a real operation than the api would error.

Reply:
My understanding is operation name is client defined; it has to correspond to the operation name used in the query but that's also client defined. So it's helpful for debugging but not for an audit.

Reply:
Ah okay, I see. Thanks.